### PR TITLE
Include contents list for html publications when printing

### PR DESCRIPTION
**Pull request by russellthorn**
... So that users are more confident about what's in the document and it's structure when they read it offline.

• display: block!important declaration used to override the !important declaration set on hiding the body nav
• remove unneccessary margins for better alignment
• bold the numbering to match desktop styles
• reduce padding around nav li's to reduce wasted printed vertical space but maintain clarity

https://www.agileplannerapp.com/boards/105200/cards/8667

### CODE REVIEW

File: app/assets/stylesheets/frontend/print/_html-publication.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.html-publications-show {
+  nav.in-page-navigation {
+    display: block!important;
```
Comment (edds) [inline app/assets/stylesheets/frontend/print/_html-publication.scss]: Is there a way to achieve this without an `important`? Using `important` is a last resort we should really avoid.

Comment (edds) [inline app/assets/stylesheets/frontend/print/_html-publication.scss]: Just seen the comment in the commit, sorry. Do you think it would be hard to remove the previous use of `important`?

Comment (russellthorn) [inline app/assets/stylesheets/frontend/print/_html-publication.scss]: That's ok. Not hard to remove the other !important. I think it would be very time consuming checking all the possible knock on effects as it looks like it could effect a lot of things. I did consider removing the other !important but decided on doing it this way for this quick fix to show the html pubs content list.

This is the what it's overriding -
body #global-user-location, body .related-positioning, header.page-header div h1 span, body .actions, body nav, #local-locator-form, #location-loading, #get-started, .finding_location, .found_location, .loading_location, .location_error, #global-browser-prompt, #user-satisfaction-survey, .report-a-problem-toggle, .report-a-problem-container, .show-all-parts {
display: none !important;
}

Comment (edds) [inline app/assets/stylesheets/frontend/print/_html-publication.scss]: Gosh [that selector](https://github.com/alphagov/static/blob/master/app/assets/stylesheets/helpers/_core-print.scss#L224-L243) seems a little much. Trying to fix that seems outside the scope of this change.

**Comment by edds** [comment]
:+1:
